scanner.go

```diff
@@ -359,3 +378,63 @@ func (s *scanner) closeRegionScanner() {
 	}
 	s.curRegionScannerID = noScannerID
 }
+
+// renews a scanner by resending scan request with renew = true
+func (s *scanner) renew() error {
+	if err := s.rpc.Context().Err(); err != nil {
+		return err
+	} else {
+		rpc, err := hrpc.NewScanRange(s.rpc.Context(),
+			s.rpc.Table(),
+			s.startRow,
```

Comment:
Ughh... I just realized this doesn't work because gohbase doesn't allow you to send the same rpc twice. 
So, the way to do this would be to pass in the startRow.

Reply:
Implemented this, let me know if you have any feedback!